raft.spec

```diff
@@ -7,7 +7,7 @@
 %bcond_with use_release
 
 Name:		raft
-Version:	0.7.1
+Version:	0.7.2
```

Comment:
I understand there is a conflict with pull/42. Will rebase if PR [42](https://github.com/daos-stack/raft/pull/42) lands first